fypp-gfortran.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import sys
+import subprocess
+from pathlib import Path
+
+# Read the system arguments.
+args = sys.argv[1:]
+
+# Get the filenames with .fypp extension and convert to string.
+source_file = [arg for arg in args if arg.endswith(".fypp")]
+output_file = [arg for arg in args if arg.endswith(".o")]
+
+if len(source_file) != len(output_file):
+    subprocess.run(["gfortran"] + args, check=True)
```

Comment:
While I know that it is a temporary solution, does this mean that only gfortran is supported currently with `fypp` files?

Reply:
We could pick up the  FC env variable here, but keep in mind that the name of the script is used by fpm to identify the compiler. For ifort we need a fypp-ifort.py with almost identical content.